File: ProgrammCourses/Sem4/java-advanced/solutions/java-solutions/info/kgeorgiy/ja/levitskii/i18n/statisitics/Tester.java
package info.kgeorgiy.ja.levitskii.i18n.statisitics;

import org.junit.runner.JUnitCore;

public class Tester {
    public static void main(String[] args) {
        //by kgeorgiy's Tester
        final long start = System.currentTimeMillis();


        new JUnitCore().run(TestTextAnalyze.class);

        System.out.println("============================");
        final long time = System.currentTimeMillis() - start;
        System.out.printf("OK %s in %dms %n", TestTextAnalyze.class.getName(), time);
    }
}

File: ProgrammCourses/Sem5/mt/Lab2Parser/src/test/Tester.kt
package test

import org.junit.runner.JUnitCore
import kotlin.reflect.KClass

fun runTest(clazz: KClass<out Any>) : Long {
    val start = System.currentTimeMillis()
    JUnitCore().run(clazz.java)

    println("============================")
    val time = System.currentTimeMillis() - start

    System.out.printf("OK %s in %dms %n", clazz.java.name, time)
    return time
}

fun main() {
    val time1 = runTest(TestLexicalAnalyzer::class)
    val time2 = runTest(TestParser::class)

    println("====================================================================================")
    System.out.printf("OK %s in %dms %n", "Full Tests", time1 + time2)
}
